### DISCUSSION TITLE
Cannot load URDF file

### DISCUSSION THREAD

**Discussion by Mark799** [bulletphysics/bullet3#3987] [Q&A]
When I try to run an example file, like the one below, I keep getting the error that the bunny won't load.
https://github.com/bulletphysics/bullet3/blob/master/examples/pybullet/examples/load_soft_body.py

The plane, samurai, cube, table, and sphere2 can all be imported. But objects like the bunny can't be loaded.

I've run the code in a Jupiter Notebook (Python 3.6), as well as on Visual Studio (Python 3.7) on Windows 10. I've installed PyBullet using pip install pybullet.

What could be the cause of this, and how can I fix it?

**Comment by sven1904** [comment]
Did you check if `bunny.obj` exists in the path that is returned by `pybullet_data.getDataPath()`?

**Comment by erwincoumans** [comment]
Indeed, bunny.obj isn't added to the right data folder.
Fixed in this PR:
https://github.com/bulletphysics/bullet3/pull/4010
